integration-tests/pom.xml

```diff
@@ -78,6 +78,13 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <!-- For reflection on S3OutputStream in tachyon.master.journal.JournalWriter -->
```

Comment:
For `S3OutputStream`, I'll try to determine the stream is for S3 without reflection on the stream.